src/internal/connector/data_collections.go

```diff
@@ -124,6 +136,29 @@ func verifyBackupInputs(sels selectors.Selector, userPNs, siteIDs []string) erro
 	return nil
 }
 
+func checkServiceEnabled(
+	ctx context.Context,
+	gs graph.Servicer,
+	service path.ServiceType,
+	resource string,
+) (bool, error) {
+	if service == path.SharePointService {
+		// No "enabled" check required for sharepoint
+		return true, nil
+	}
+
+	_, info, err := discovery.User(ctx, gs, resource)
```

Comment:
nit: We could probably limit the check to only the service that one is trying to backup instead of both exchange and onedrive.
Other than that, we should be able to follow something similar for OneDrive.

Reply:
Going to leave the check as is for now and make that change in a follow-up (need to get this fix in first)